src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -24,13 +30,27 @@ private static string GetFileNameFromResourceName(string s)
             return parts[parts.Length - 2] + "." + parts[parts.Length - 1];
         }
 
-        public int CreateEmptyProject()
+        public int CreateEmptyProject(Language language)
```

Comment:
Well, attacker won't get anything they can't already get so I guess it's ok :) Whitelist sounds good to me (some mappings csharp->c# or reverse sound good too)